_posts/2017-01-17-Oredev2016.md

```diff
@@ -0,0 +1,695 @@
+---
+layout: post
+authors: [kevin_van_houtte]
+title: 'Øredev 2016'
+image: /img/oredev/oredevonwhite.png
+tags: [Conference]
+category: Conferences
+comments: true
+---
+
+When arrived in Malmö city, we were welcomed with an exploration tour of the third biggest city in Sweden.
+The next day Øredev 2016 officially started!
+It was noticeable that there was a healthy combination for everyone: "Microservices", "Docker", "UX", "Management", etc...
+All these buzzwords were applied in various talks.
+This blog post will be about the talks I favored and where I learned the most from and still use in my nowadays projects.
```

Comment:
Would rephrase:
This blog post will be about the talks I favored ('favoured' if the post is to be written in British English) and the extra knowledge I gained which I now get to apply in my nowadays projects.